eng/pipelines/templates/jobs/vmr-synchronization.yml

```diff
@@ -22,8 +18,8 @@ parameters:
   default: false
 
 jobs:
-- job: ${{ parameters.jobName }}
-  displayName: Synchronize VMR's ${{ parameters.vmrBranch }}
+- job: Synchronize_VMR
+  displayName: Synchronize dotnet/dotnet
```

Comment:
I'd leave the branch name in the job name, it's handy even for `release/` branches etc

Reply:
```suggestion
  displayName: Synchronize VMR's ${{ parameters.vmrBranch }}
```